res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -1085,14 +1052,28 @@ NumarkMixtrack3.init = function(id, debug) {
     print("");
 
     // Set soft-takeover for all Sampler volumes
-    for (i = engine.getValue("[Master]", "num_samplers"); i >= 1; i--) {
+    for (var i = engine.getValue("[Master]", "num_samplers"); i >= 1; i--) {
         engine.softTakeover("[Sampler" + i + "]", "pregain", true);
     }
 
-    for (i = engine.getValue("[Master]", "num_decks"); i >= 1; i--) {
+    for (var i = engine.getValue("[Master]", "num_decks"); i >= 1; i--) {
```

Comment:
FYI, in the old version of JS that Mixxx still uses (and in modern JS using `var`), variables are scoped to functions, not blocks as you might expect. You don't need to redeclare `i` for each loop. Just something to be aware of if you encounter puzzling bugs with one loop messing up the behavior of the next.